pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -18,6 +18,8 @@ type KeycloakRealmSpec struct {
 	// +kubebuilder:validation:Required
 	Realm *KeycloakAPIRealm `json:"realm"`
 	// A list of overrides to the default Realm behavior.
+	// +listType=map
+	// +listMapKey=identityProvider
```

Comment:
A better solution is to make this `+listType=atomic`. We don't really care about merging this list.

Reply:
Done.

Reply:
Thank you!